eva_submission/eload_ingestion.py

```diff
@@ -372,3 +372,16 @@ def run_variant_load_workflow(self):
     @cached_property
     def valid_vcf_filenames(self):
         return list(self.project_dir.joinpath(project_dirs['valid']).glob('*.vcf.gz'))
+
+    @cached_property
+    def vcf_merge_type(self):
+        file_to_sample_names = {}
+        # retrieve all the sample_names from the VCF files
+        for file_path in self.valid_vcf_filenames:
+            file_to_sample_names[file_path] = get_samples_from_vcf(file_path)
+        # Check that all the samples are the same and in the same order to enable horizontal merging
+        sample_info = compare_sample_sets(file_to_sample_names.values())
+        if sample_info == 'unique sample sets':
+            return 'horizontal merging'
+        elif sample_info == 'single set':
+            return 'vertical merging'
```

Comment:
personally i prefer explicitly returning `None` (for methods that can return non-none values) - especially in this case as the other options for `sample_info` are potentially meaningful

Reply:
I've added the explicit `None`